docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -0,0 +1,115 @@
+# How to convert a changelog to a table
+
+## Context
+
+You have a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog stream into a table. This is broadly called *materializing* a table.
+
+## In action
+
+```sql
+CREATE TABLE t1 AS
+    SELECT k,
+           LATEST_BY_OFFSET(v1) AS v1,
+           LATEST_BY_OFFSET(v2) AS v2,
+           LATEST_BY_OFFSET(v3) AS v3
+    FROM s1
+    GROUP BY k
+    EMIT CHANGES;
+```
+
+## Materializing a changelog stream
+
+In ksqlDB, you derive new tables by aggregating other streams and tables. To create a table that reflects the latest values for each key, use the `LATEST_BY_OFFSET` aggregation.
+
+Begin by telling ksqlDB to start all queries from earliest point in each topic.
+
+```sql
+SET 'auto.offset.reset' = 'earliest';
+```
+
+Make a stream `s1` with four columns. `k` represents the key of the table. Rows with the same key represent information about the same entity. `v1`, `v2`, and `v3` are various value columns.
+
+```sql
+CREATE STREAM s1 (
+    k VARCHAR,
+    v1 INT,
+    v2 VARCHAR,
+    v3 BOOLEAN
+) WITH (
+    kafka_topic = 's1',
+    partitions = 1,
+    value_format = 'avro',
```

Comment:
What's the reasoning behind using Avro in the examples rather than JSON? Avro requires Schema Registry so it's more pieces to get off the ground.

Reply:
We're encouraging using Avro everywhere just so people can get the benefits of Schema Registry. Im all in favor for fewer moving parts, but I think this one is important.